lib/platform/bitbucket-server/index.ts

```diff
@@ -289,17 +289,6 @@ export async function getPr(
   pr.hasReviewers = is.nonEmptyArray(pr.reviewers);
   pr.version = updatePrVersion(pr.number, pr.version);
 
-  if (pr.state === PrState.Open) {
-    const mergeRes = await bitbucketServerHttp.getJson<{
-      conflicted: string;
-      canMerge: string;
-    }>(
-      `./rest/api/1.0/projects/${config.projectKey}/repos/${config.repositorySlug}/pull-requests/${prNo}/merge`,
-      { useCache: !refreshCache }
-    );
-    pr.canMerge = !!mergeRes.body.canMerge;
-  }
```

Comment:
This endpoint only reflects git mergeability (which we check ourselves using git) as well as status checks, which we check separately.

Reply:
Doesn't it also reflect required review?

Reply:
I think we may as well just attempt the automerge instead of an extra API call every PR, what do you think?

Reply:
ok